policy/modules/apps/cdrecord.te

```diff
@@ -113,3 +132,98 @@ tunable_policy(`use_nfs_home_dirs',`
 optional_policy(`
 	resmgr_stream_connect(cdrecord_t)
 ')
+
+########################################
+#
+# Cdripper (e.g. cdda2wav) local policy
+#
+
+allow cdripper_t self:capability { ipc_lock sys_nice setgid setuid dac_override sys_rawio };
+allow cdripper_t self:process { getcap getsched setcap setrlimit setsched sigkill };
+allow cdripper_t self:fifo_file rw_fifo_file_perms;
+allow cdripper_t self:tcp_socket create_socket_perms;
+allow cdripper_t self:udp_socket create_socket_perms;
+allow cdripper_t cdripper_exec_t:file execute_no_trans;
+
+corecmd_exec_bin(cdripper_t)
+corecmd_exec_shell(cdripper_t)
+
+corenet_tcp_connect_all_unreserved_ports(cdripper_t)
+
+dev_list_all_dev_nodes(cdripper_t)
```

Comment:
Not your fault but:

This interface does not make sense to me (at least its name, i have not looked it up so i do not know what it implies).  "List" is a directory permission. There is only one directory type in "/dev" which is "device_t" so to say list all dev nodes seems weird. You don't "list dev nodes"

I think this should probably have been splitted in at least two interfaces with saner names, for example:
```
dev_list(cdripper_t)
dev_getattr_all_dev_nodes(cdripper_t)
```
maybe even a `dev_read_lnk_files(cdripper_t)` added.

Reply:
NACK. Already there and in common use.